test/ctia/auth/jwt_test.clj

```diff
@@ -38,66 +39,99 @@
     (is (instance? ctia.auth.jwt.JWTIdentity
                    (get-in response-jwt [:body :identity])))))
 
+(deftest assets-scope-test
+  (testing "Assets scope and capabilities"
+    (helpers/with-properties ["ctia.auth.entities.scope" "global-intel"
+                              "ctia.auth.assets.scope" "asset-intel"]
+      (helpers/fixture-ctia-with-app
+       (fn [app]
+         (let [{:keys [get-in-config]} (helpers/get-service-map app :ConfigService)]
+           (is (= "global-intel" (sut/entity-root-scope get-in-config)))
+           (is (= #{:create-asset :create-asset-mapping :create-asset-properties
+                    :create-target-record :delete-asset :delete-asset-mapping
+                    :delete-asset-properties :delete-target-record :list-asset-mappings
+                    :list-asset-properties :list-assets :list-target-records :read-asset
+                    :read-asset-mapping :read-asset-properties :read-target-record :search-asset
+                    :search-asset-mapping :search-asset-properties :search-target-record}
+                  (sut/scope-to-capabilities (sut/assets-root-scope get-in-config) get-in-config))
+               "By default asset capabilities enabled on global-intel scope")))))
+    (testing "Asset capabilities explicitly disabled"
+      (helpers/with-properties ["ctia.auth.entities.scope" "global-intel"
+                                "ctia.auth.assets.scope" "no-asset-intel"]
```

Comment:
This configuration is only here to specify the value of the scope that provides asset capabilities, "asset-intel" being the default value for such a scope. So such a conf would specify that the scope accepted to have asset capabilities would be `no-asset-intel`

Reply:
I tried addressing your comments in https://github.com/threatgrid/ctia/pull/1078/commits/fc313a228753acdf01f680f85bdcb7318fc9d065

And Guillaume's in https://github.com/threatgrid/ctia/pull/1078/commits/a76efff22ee5110ed1ac73fa60dbffc68b9468f4

Would that do? Let me know if I missed anything else.